webapp/app.py

```
@@ -779,5 +915,203 @@ def language(lang_code):
+        }
+
+        stats["total"] += 1
+        if won:
+            stats["wins"] += 1
+            if isinstance(attempts, int) and 1 <= attempts <= 6:
+                stats["distribution"][str(attempts)] += 1
+        else:
+            stats["losses"] += 1
+
+        _save_word_stats(lang_code, day_idx, stats)
+        return "", 200
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)